File: Domain/BusService.cs
namespace Model
{
    public class BusService
    {
        public Guid? BusServiceId { get; set; }
        public string? RegistrationNumber { get; set; }
        public string? Name { get; set; }

        public string? Address { get; set; }

        public string? Phone { get; set; }

        public string? Email { get; set; }

        public string? Password { get; set; }

        public List<Schedule>? Schedules { get; set; }
        public List<Bus>? Buses { get; set; }

    }
}

File: ReservedYouFrontEnd/src/app/models/BusService.ts
export class BusService {
  busServiceId: string;
  registrationNumber: string;
  name: string;
  address: string;
  email: string;
  phone: string;
  password: string;

  constructor(
    bsId: string,
    regNo: string,
    name: string,
    address: string,
    email: string,
    phone: string,
    password: string
  ) {
    this.busServiceId = bsId;
    this.registrationNumber = regNo;
    this.name = name;
    this.address = address;
    this.email = email;
    this.phone = phone;
    this.password = password;
  }
}
